src/vim-webgl-viewer/camera/cameraMovementDo.ts

```diff
@@ -1,4 +1,7 @@
-import { Camera } from './camera'
+/**
```

Comment:
The file name and class "cameraMovementDo" gives me pause. Would the name "cameraMovementAction" be more suitable to replace the "Do" term?

Reply:
It's to match CamaeraMovementLerp
The api is camera.do().move() and camera.lerp().move()